Dao cont prettified

tharmaman opened the pull request.

mcarcaso committed:
Initial checkin of new daocontroller and some minor fixes along the way.

mcarcaso committed:
Merge branch 'master' into DAOCont

mcarcaso committed:
Merge branch 'master' into DAOCont

mcarcaso committed:
Merge branch 'master' into DAOCont

mcarcaso committed:
Make MapView's data a Map

mcarcaso committed:
Make some of the new comics pieces more compatible with java.

mcarcaso committed:
Remove the controllerMode logic from FObjectArrayView because the DetailView it's given to can figure that all out on its own.

mcarcaso committed:
Merge with master

mcarcaso committed:
Undo mapview change

mcarcaso committed:
Merge branch 'master' into DAOCont

mcarcaso committed:
Some updates to the new dao controllers. Still very incomplete.

mcarcaso committed:
Merge branch 'master' into DAOCont

mcarcaso committed:
Use new dao in country

Commit:
Got started with pretty stuff

Commit:
Working on some thangs

Commit:
added icons

Commit:
Added up the final touches

Commit:
Removed unneccessary tag

Commit:
Made the table headers match logan's designs

Commit:
Got the tings to show up

Commit:
Got started on adding the qcannedqueries

Commit:
used a choice view j stead

Commit:
Created a TabChoiceView and passed it into the comic instead of a radioview

Commit:
Fixed up merge conflict

Commit:
Added DAOMenu2 to the files

Commit:
Changed export icon and got started with conditional canned queries

Commit:
Merge branch 'master' into DAOContPrettified

Commit:
Touched up some stuff

Commit:
Added some more tings

Commit:
reset the browseview condition to > 1

Commit:
Did some more tings

Commit:
Added in the conditional showing depending on canned queries

Commit:
Added in conditional rendering of top bar

Commit:
Added in an extra comment

Commit:
Made the tabview omg i love jesus

Commit:
Added a comment to figure out a better way to do the width later on

Commit:
Added in a potential for editing the defaultTabWidth

Commit:
Got the icon choice view working

mcarcaso committed:
Merge with master

Commit:
Got the icon choice view working

Commit:
Merge branch 'DAOCont' into DAOContPrettified

Commit:
Chaned disabled icon to use opacity

Commit:
Removed unnecessary console log

Commit:
Wired up the top part of the detail view

Commit:
Wired up grabbing the canned queries

Commit:
Added new tings to ting

Commit:
fixed weird conflicts

Commit:
Fixed up merge conflicts

Commit:
Wired up create

Commit:
Hooked up the export

Commit:
Restored country, enabled export, fixed up the popping back of stack view and the rendering of primary action:

Commit:
Playred with tings

Commit:
Wired up the browseViews to the DAOController through axioms

Commit:
Added in the fix for expressions v factories

Commit:
Fixed up default tab choice view spacing

Commit:
Fixed merge conflicts

Commit:
Removed the logs cronjobs and notifications from the git cache

Commit:
Refactored some tings and took out useless code

Commit:
Fixed up rendering of create button

Commit:
Merge branch 'Viewfixes' into DAOContPrettified

Commit:
Cleaned up comments

Commit:
Moved the browseView outside of the browserView

Commit:
Binded the browse view in the controller

Commit:
Floated the icons to the right

Commit:
Fixed the iconchoice and tabchoiceviews, left a comment in the radioview and wired up the icons

Commit:
Added the bottom border of the view

Commit:
Made a temporary fix for the scrollbar-container height

Commit:
Made the dynamicHeight for the ScrollTable optional

Commit:
Switched up the disabling the icon

Commit:
Merge branch 'master' into DAOContPrettified

mcarcaso commented on src/foam/nanos/auth/Country.js:
```diff
@@ -12,6 +12,51 @@ foam.CLASS({
 
   ids: ['code'],
 
+  axioms: [
```
Exclude these

mcarcaso commented on src/foam/u2/view/IconChoiceView.js:
```diff
@@ -0,0 +1,101 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc. All Rights Reserved.
```
Use this
```
/**
 * @license
 * Copyright 2019 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */
```
here and in all other new files.

mcarcaso commented on src/foam/u2/search/Toolbar.js:
```diff
@@ -0,0 +1,32 @@
+foam.CLASS({
```
```
/**
 * @license
 * Copyright 2019 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */
```

kgrgreer commented on src/foam/nanos/menu/DAOMenu2.js:
```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
```
2019

Commit:
cleaned up test code from country

Commit:
changed copyright to 2019

Commit:
Fixed up the license

mcarcaso commented on src/cronjobs:
```diff
@@ -1,3 +0,0 @@
-p({"class":"foam.nanos.cron.Cron","minute":1,"hour":-1,"dayOfMonth":-1,"month":-1,"dayOfWeek":-1,"scheduledTime":"2017-11-03T15:01:00.0Z","id":"sessionExpiry","description":"Remove expired sessions.","lastRun":"2017-11-03T14:21:09.231Z","server":true,"scheduled":false,"code":"EXPIRY_AGE = 8*3600*1000; /* 8 hours */ dao = x.get(\"localSessionDAO\");\ndao.where(foam.mlang.MLang.LT(foam.nanos.sessio ...
```
I don't know if we need any of cronjobs. might be safest not to delete this.

mcarcaso commented on src/foam/nanos/controller/ApplicationController.js:
```diff
@@ -106,7 +106,7 @@ foam.CLASS({
       letter-spacing: 0.2px;
       color: #373a3c;
-      background: %BACKGROUNDCOLOR%;
+      background: #f5f7fa;
```
undo

mcarcaso commented on src/foam/u2/ActionView.js:
```diff
@@ -166,18 +166,18 @@ foam.CLASS({
-      color: white;
+      border: 2px solid transparent;
+      background-color: transparent;
+      color: #5e6061;
```
Shouldn't this be something from the look and feel?

mcarcaso commented on src/foam/u2/ActionView.js:
```diff
@@ -166,18 +166,18 @@ foam.CLASS({
-    ^tertiary:focus {
-      box-shadow: 0 1px 2px 0 rgba(22, 29, 37, 0.1), inset 0 1px 0 1px rgba(255, 255, 255, 0.06);
+    ^tertiary:focus:not(:hover) {
+      border-bottom-color: #4a33f4;
```
Same here. Probably should be part of a theme.

Commit:
Fixed up some more tings

Commit:
added cronjobs

Commit:
cronjobs back

Commit:
Added back into the gitignore

Commit:
Had to add a newline to the end

Commit:
Added newlines to the end

Commit:
Added better comments and themed colours instead to the tertiary action view

Commit:
Added a theme for the tertiary button

mcarcaso committed:
A bunch of cleanup:
- Move around some css to be on the views that actually care about it.
- Try to use less manual targetting of css classes.
- Fix up some code style (put more things on new lines)

mcarcaso committed:
Fix alt-view

Commit:
Merge branch 'master' into DAOContPrettified

Commit:
Merge branch 'master' into DAOContPrettified

tharmaman committed:
Merge pull request #5 from mcarcaso/DAOContPrettified

First pass at reviewing new dao controller PR

Commit:
Got started on mike's revisions
